lib/layout/list_view.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:ensemble/framework/widget/widget.dart';
+import 'package:ensemble/layout/layout_helper.dart';
+import 'package:ensemble/layout/templated.dart';
+import 'package:ensemble/page_model.dart';
+import 'package:ensemble/util/utils.dart';
+import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart' as flutter;
+
+// ignore: must_be_immutable
+class ListView extends BoxLayout {
+  static const type = 'ListView';
+  ListView({Key? key}) : super(key: key);
+
+  @override
+  bool isVertical() {
+    return true;
+  }
+}
+
+abstract class BoxLayout extends StatefulWidget
```

Comment:
I don't understand. Is this a duplicate of the original BoxLayout? Did you mean to remove the original one?